pychatlib/client/client.py

```diff
@@ -20,13 +20,28 @@ def __init__(self):
             "slid in!",
             "showed up!",
             "joined the party!",
+            "has found the chat!",
+            "has found a new cake for us!",
```

Comment:
remove "has found a cake for us", ", do you have zircoins", "saw your face", "was rickrolled", "was exposed by aa virus" and "deleted wrong file"